lib/commands/general.js

```diff
@@ -216,6 +221,100 @@ commands.closeApp = async function () {
   await this.adb.forceStop(this.opts.appPackage);
 };
 
+commands.getDisplayDensity = async function () {
+  let out = await this.adb.shell(['getprop', 'ro.sf.lcd_density']);
+  if (out) {
+    return parseInt(out);
```

Comment:
There is a potential issue with `parseInt` when there is no radix specified. It ought to be added

```js
return parseInt(out, 10);
```